modules/axmm/src/lib.rs

```diff
@@ -106,6 +106,8 @@ pub fn init_memory_management() {
     debug!("kernel address space init OK: {:#x?}", kernel_aspace);
     KERNEL_ASPACE.init_once(SpinNoIrq::new(kernel_aspace));
     unsafe { axhal::asm::write_kernel_page_table(kernel_page_table_root()) };
+    // flush all tlb
+    axhal::asm::flush_tlb(None);
```

Comment:
Also add it in `init_memory_management_secondary`. And the flush is not required on x86_64.